test/test-build.js

```diff
@@ -39,6 +39,18 @@ function doTests(transpiler) {
     });
   });
 
+  test('Static string build', function () {
+    builder.reset();
+    builder.config({ transpiler: transpiler });
+    return builder.bundle('foo.js', {
+      fetch: function (data) {
+        if (data.name.indexOf('foo.js') !== -1) {
+          return fs.readFileSync('test/fixtures/test-tree/amd-1.js', 'utf8');
+        }
+      }
```

Comment:
The suggestion above is based on a single extension of the loader that checks for a `builder.fetch` instance function on the builder not the loader itself. That is, setting `builder.fetch = null` completely reverses the fetch hook override. I'd prefer not to provide a fallthrough unless there is a good use case to justify it?